frontend/src/components/Button/Button.jsx

```diff
@@ -44,7 +44,7 @@ export const Button3 = (props) => {
   return (
     <button
       type={props?.type || "button"}
-      className="main-btn main-btn-2 main-btn-3"
+      className={`${style["main-btn"]} ${style["main-btn-2"]} ${style["main-btn-3"]}`}
```

Comment:
before: `className="main-btn main-btn-2 main-btn-3"`

after: `className={`${style["main-btn"]} ${style["main-btn-2"]} ${style["main-btn-3"]}`}`

in this line, if you can suggest a better clean way to write multiple css classes to an element, then I ll be glad to implement that.